recipes/micronota/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pip install burrito click
```

Comment:
> Is there a way to set pip installable dep in requirements section?
I don't think so, but I see it would be useful.

I can't test it tonight, but I'd try installing and running the package in a clean environment without burrito/click to see if it actually works.